alchy/utils.py

```diff
@@ -92,12 +92,12 @@ def get_mapper_class(model, field):
     return mapper_class(getattr(model, field))
 
 
-def merge_declarative_args(base_classes, config_key):
+def merge_local_declarative_args(cls, config_key):
```

Comment:
Let's keep the name the same. I'm not sure `local` has any real context within the function itself.